extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/connection/ConnectionService.java

```diff
@@ -171,9 +210,73 @@
                     logger.debug("LDAP exception when getting protocol value.", e);
                     return null;
                 }
+                
+                // Get the default Proxy configuration
+                GuacamoleProxyConfiguration proxyConfig;
+                try {
+                    proxyConfig = confService.getDefaultGuacamoleProxyConfiguration();
```

Comment:
If `SimpleConnection` will use the default configuration from the environment if no configuration is provided, is this still needed?

Reply:
Makes sense.